docs/management_api.yml

```diff
@@ -356,6 +356,7 @@ paths:
 
   /deployments/{deployment_id}/devices:
     get:
+      deprecated: true
       operationId: List Devices in Deployment
```

Comment:
This is not enough to render a nice "deprecation message" in the Mender API docs website.
Please update the Summary adding a clear deprecation message, e.g.:
DEPRECATED: this end-point is deprecated because it doesn't support pagination and will be removed in the future, please use the `/deployments/{deployment_id}/devices/list` end-point instead.